src/helpers/airdrop.ts

```diff
@@ -1,7 +1,86 @@
-import { CONST } from 'app/consts';
+import { CONST, AirdropGoal, AirdropCarouselCardData } from 'app/consts';
 import { formatDate, getTimezoneOffset, isAfter } from 'app/helpers/date';
 
+import { formatToBtcvWithoutSign } from '../../utils/bitcoin';
+
+const i18n = require('../../loc');
+
 export const getFormattedAirdropDate = () =>
   `${formatDate(CONST.airdropDate.local(), 'DD/MM/YYYY h a')} ${getTimezoneOffset()}`;
 
 export const isAfterAirdrop = () => isAfter(new Date(), CONST.airdropDate);
+
+export const readableOrder = [
+  i18n.order.first,
+  i18n.order.second,
+  i18n.order.third,
+  i18n.order.fourth,
+  i18n.order.fifth,
+  i18n.order.sixth,
+];
+
+export const lastThresholdBeforeInfinity = 20000;
+
+export const airdropCommunityGoals: AirdropGoal[] = [
+  { threshold: 2500, value: '25000' },
```

Comment:
I would expect that the value would be a number as the threshold is, not a string

Reply:
it s only getting rendered, no manipulation and this way it can be both text and number (as string). Actually when you think about it, value abstract word, it s not pointing to and specific type, so why not string

Reply:
ok, let's keep string